src/commands/releases.rs

```diff
@@ -428,6 +430,10 @@ fn execute_set_commits<'a>(
     matches: &ArgMatches<'a>,
 ) -> Result<(), Error> {
     let version = matches.value_of("version").unwrap();
+    let default_count = matches
```

Comment:
It's unnecessary to parse it here, as it's relatable only for `--local` code path. Move it there instead.